features/git-ship/current_branch/on_feature_branch/without_open_changes/commit_message_containing_double_quotes.feature

```diff
@@ -35,3 +35,22 @@ Feature: git ship: shipping the current feature branch
     And I have the following commits
       | BRANCH | LOCATION         | MESSAGE                            | FILE NAME    |
       | main   | local and remote | message containing "double quotes" | feature_file |
+
+
+  Scenario: undo
+    When I run `git ship --undo`
+    Then it runs the Git commands
+      | BRANCH  | COMMAND                                                    |
+      | main    | git branch feature <%= sha 'feature commit' %>             |
+      |         | git push -u origin feature                                 |
+      |         | git revert <%= sha 'message containing "double quotes"' %> |
+      |         | git push origin main                                       |
+      |         | git checkout feature                                       |
+      | feature | git checkout main                                          |
+      | main    | git checkout feature                                       |
+    And I end up on the "feature" branch
+    And I have the following commits
+      | BRANCH  | LOCATION         | MESSAGE                                     | FILE NAME    |
+      | main    | local and remote | message containing "double quotes"          | feature_file |
+      |         |                  | Revert "message containing "double quotes"" | feature_file |
+      | feature | local and remote | feature commit                              | feature_file |
```

Comment:
I don't think this needs an undo test. Its focused on just dealing with a commit message that has double quotes